package.json

```diff
@@ -47,6 +48,8 @@
     "react-router-dom": "^5.0.1",
     "react-test-renderer": "^16.9.0",
     "redux": "^4.0.4",
+    "redux-mock-store": "^1.5.3",
```

Comment:
i think that this package will be used only in tests, so probably it should go to `devDependencies`